### PR TITLE
Update pino 

### PR DESCRIPTION
**Pull request by Kevan-Y**
Fixes [#2499](https://github.com/Seneca-CDOT/telescope/issues/2499)

Fix: 
- Update Pino
- Fix deprecated prettyPrint 
- Switch from express-pino-logger to pino-http
- Use pino-pretty instead of pino-colada
- Standardize Dependencies 

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,15 +28,15 @@
...
-    "express-pino-logger": "^6.0.0",
     "helmet": "4.6.0",
     "http-errors": "^1.8.0",
     "ioredis": "^4.27.1",
     "ioredis-mock": "^5.5.5",
     "jsonwebtoken": "^8.5.1",
     "node-fetch": "^2.6.1",
-    "pino": "^6.11.3",
-    "pino-colada": "^2.1.0"
+    "pino": "^7.6.4",
```
Comment (humphd) [inline package.json:37]: We have a weird mix of pinned and `^` versions in this file.  In a follow-up, we should standardize (probably on pinned like Telescope).  cc @DukeManh.

File: src/app.js [hunk 2]
```diff
@@ -2,15 +2,14 @@ const express = require('express');
 const cors = require('cors');
 const helmet = require('helmet');
 const createError = require('http-errors');
-const expressPino = require('express-pino-logger');
-
+const pinoHttp = require('pino-http');
 const logger = require('./logger');
```
Comment (humphd) [inline src/app.js:6]: Add a blank line between external module imports and relative modules in the tree.

File: src/logger.js [hunk 3]
```diff
@@ -2,8 +2,12 @@ const pino = require('pino');
 
 // Pretty debug logging
 let logger = pino({
-  prettyPrint: {},
-  prettifier: require('pino-colada'),
+  transport: {
+    target: 'pino-pretty',
```
Comment (humphd) [inline src/logger.js:6]: We should only do this `transport` option if we're in `debug` log level.

**Review by humphd — CHANGES_REQUESTED** [review]

File: src/logger.js (hunk 3 above)
Comment (Kevan-Y) [reply src/logger.js:6]: Do we have env in satellite to run in `debug`  log level?

Comment (humphd) [reply src/logger.js:6]: Satellite honours whatever env vars are available.  We don't set them there, we set them in the Docker containers that run Satellite.  So you can use `process.env.LOG_LEVEL` and not worry about how it gets set.

**Review by humphd — APPROVED** [review]

**Comment by humphd** [comment]
We should do a release of Satellite this week to pick up this change and the node-fetch update that's coming.

cc Sheriffs: @AmasiaNalbandian, @DukeManh

**Comment by DukeManh** [comment]
@humphd, I agree

**Pull request merged**